lib/membrane_caps_audio_raw.ex

```diff
@@ -283,6 +283,27 @@ defmodule Membrane.Caps.Audio.Raw do
     end
   end
 
+  @doc """
+  Returns a binary which corresponds to the silence during the given interval
+  of time in given caps' fromat
+
+  ## Examples:
+  The following code generates one second of the silence for the given caps
+
+    iex> alias Membrane.Caps.Audio.Raw, as: Caps
+    Membrane.Caps.Audio.Raw
```

Comment:
This line can be removed, results are necessary only when they need to be tested